src/field_frame.h

```diff
@@ -177,7 +178,7 @@ class FieldFrame : public wxDocChildFrame {
 
     FieldFrameControls* mControls;
     FieldCanvas* mCanvas;
-    FieldBrowser* mFieldBrowser;
+    FieldThumbnailBrowser* mFieldThumbnailBrowser;
```

Comment:
Looks like we're leaking this object right now. Maybe we should consider making this a `unique_ptr`? I know that's not necessarily in the scope of these changes, so maybe that's worth a different issue. The other raw pointers around this one could probably use rework too.

Reply:
Actually, these aren't leaked.  Because they are wxWindows they get managed by the larger frame.  This is a confusing point of wxWidgets (and also QT) that the memory management is a little opaque.

Reply:
Ah okay; so adding a `unique_ptr` would actually be a pretty bad idea here...

Should we make it a reference or anything else? Do you think that a pointer is the best selection for an object managed by `wxWidgets`?

Reply:
A reference doesn't work because it's a dynamic object.  It might be better to make a helper function to extract it from the frame...

Reply:
Is it really dynamic? I only see it set up only once, in the constructor. (I'm taking 'dynamic' to mean: it changes over the lifetime of `FieldFrame`; let me know if that's wrong).

That being said, creation of this object does require arguments that are set up across multiple statements. Even if it is possible to convert this to a reference, the change would be nontrivial, and probably not worth doing here. I'll go ahead and close this up.